### ISSUE TITLE
ReplaceReturns transformer causes fatal error on Mac

### ISSUE THREAD

**Issue by westadejl** [MarketSquare/robotframework-tidy#281]
Hey,

When attempting to run the ReplaceReturns transformer (`robotidy --transform ReplaceReturns src`), I received a fatal error, which suggested that I open an issue, so here I am!

I'm running Robot Framework 5.0 & Python 3.10.

Traceback can be found below:

`Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/bin/robotidy", line 8, in <module>
    sys.exit(cli())
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/decorators.py", line 63, in wrapper
    raise err
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/decorators.py", line 46, in wrapper
    return func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/cli.py", line 391, in cli
    status = tidy.transform_files()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/app.py", line 64, in transform_files
    diff, old_model, new_model = self.transform(model)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/app.py", line 84, in transform
    transformer.visit(model)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robot/parsing/model/visitor.py", line 62, in visit
    return visitor(node)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/ast.py", line 486, in generic_visit
    value = self.visit(value)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robot/parsing/model/visitor.py", line 62, in visit
    return visitor(node)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/ast.py", line 486, in generic_visit
    value = self.visit(value)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robot/parsing/model/visitor.py", line 62, in visit
    return visitor(node)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/transformers/ReplaceReturns.py", line 66, in visit_Keyword
    return_stmt = create_statement_from_tokens(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/robotidy/utils.py", line 308, in create_statement_from_tokens
    return statement([indent, Token(statement.type), *tokens])
AttributeError: 'NoneType' object has no attribute 'type'
Fatal exception occurred. You can create an issue at https://github.com/MarketSquare/robotframework-tidy/issues . Thanks!`

If you need any additional information, please let me know.

Best,
Tadej

**Comment by westadejl** [comment]
The issue was my virtual environment not updating Robot Framework to version 5.0. It stayed on 4.1.3.

Once I successfully updated to Robot Framework 5.0, the issue disappeared.

**Comment by bhirsz** [comment]
Hi, thank for you the bug issue - actually I need to think about this. ``ReplaceReturns`` is default transformer (meaning if you just run robotidy . it will be executed) but it works only for RF >=5 (it's automatically disabled for RF 4.*). However now I see it's possible to overwrite this by using ``--transform`` option. This happened in your case. I believe it should still not be possible to run the ``ReplaceReturns`` transformer and when trying to run it with ``--transform`` or ``enabled=True`` options Robotidy should display something along:
```
X transformer requires Robot Framework X but you have Y installed. Upgrade Robot Framework installation if you want X transformer to work.
```

**Comment by bhirsz** [comment]
Released in Robotidy 2.2.
